curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9282cd547785258cecf2.md

```diff
@@ -20,13 +20,13 @@ assert.match(code, /taskData\.forEach\(\s*\(\s*\{\s*(?:id\s*,\s*title\s*,\s*date
 You should use arrow syntax and attach `innerHTML` to `tasksContainer`.
 
 ```js
-assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*(\(\s*tasksContainer\.innerHTML\s*\))?/)
+assert.match(code, /taskData\.forEach\(\(\{.*\}\)\s*=>\s*(\s*\{\(\s*tasksContainer\.innerHTML\s*\)\s*\})?/)
```

Comment:
Based on the recent conversation let's update this test to allow for both valid solutions 
https://github.com/freeCodeCamp/freeCodeCamp/pull/52988#issuecomment-1879834476